### ISSUE TITLE
Mode 1000 (kanji mode) not yet implemented

### ISSUE THREAD

**Issue by KyleMaas** [piderman314/bardecoder#51]
When scanning certain barcodes, I get

```
Mode 1000 not yet implemented.
```

https://github.com/piderman314/bardecoder/blob/e1fcff7f4975c8f9cc430b61b0a0ac29a4b1dd78/src/decode/qr/data.rs#L16

Not a huge deal - I don't encounter these much - but it would probably be nice to implement kanji mode eventually.

**Comment by piderman314** [comment]
Indeed, I have never seen these before. Since I have zero experience with JIS I don't think I'll spend the effort until someone has an actual usecase.
